sr_robot_msgs/msg/MSTSensor.msg

```diff
@@ -7,5 +7,5 @@ int16 magnetic_induction_y
 # Magnetic induction along z axis in gauss
 int16 magnetic_induction_z
 
-# PCB temperature in 1/100 degrees Celsius (100 = 1 degree Celsius)
-int16 temperature
+# raw taxel temperature
+uint16 temperature
```

Comment:
Why not an int? Does the range not extend below zero degrees celsius?

Reply:
Previously I was not yet converting the temperature to Celsius. However, this message does not exist anymore, and instead we will be using @bdenoun's message MST.msg (previously named MSTReading.msg). `temperature` is defined as an array of float32